boxes/static/javascripts/box_create.js

```diff
@@ -1,13 +1,36 @@
 $(document).ready(function() {
-    $("#id_expires_at").datetimepicker({
-      format: "Y-m-d H:i"
-    });
+  $("#id_expires_at").datetimepicker({
+    format: "Y-m-d H:i"
+  });
+
+  $("#id_never_expires").on("change", function(){
+    if ($(this).is(":checked")){
+      $("#id_expires_at").prop("disabled", true).val("");
+    } else {
+      $("#id_expires_at").prop("disabled", false);
+    }
+  });
 
-    $("#id_never_expires").on("change", function(){
-      if ($(this).is(":checked")){
-        $("#id_expires_at").prop("disabled", true).val("");
-      } else {
-        $("#id_expires_at").prop("disabled", false);
+  $("#box-create-form-js").on("submit", function(){
+    var $this = $(this);
+    $.ajax({
+      url: $this.attr("action"),
+      method: $this.attr("method"),
+      data: $this.serialize()
+    }).done(function(data){
+      document.location = data.location
+    }).fail(function(data){
+      var fields = ["__all__", "name", "description", "max_messages",
+                    "expires_at", "never_expires"]
```

Comment:
Missing semicolon? (I know they're optional, but don't know your opinion about keeping this consistent. I guess those omissions are caused by your pythonic ways :stuck_out_tongue: